seed-server/network-connection.sh

```diff
@@ -82,16 +102,49 @@ sed -e "s/\${SERVICE_NETWORK_ADDRESS}/${service_network_address}/" \
     -e "s/\${POD_NETWORK_NETMASK}/${pod_network_netmask}/" \
     /client-config-dir/vpn-shoot-client.template > /client-config-dir/vpn-shoot-client
 
-if [[ ! -z "$node_network" ]]; then
-    for n in $(echo $node_network |  sed 's/[][]//g' | sed 's/,/ /g')
+if [[ -n $is_ha ]]; then
+    for ((i=0; i < 2; i++))
```

Comment:
Should this not be more generic? It seems this hardcodes the two `vpn-shoot` clients.

Reply:
Thanks! Fixed.